test/feature/steps/hooks.js

```diff
@@ -0,0 +1,18 @@
+const { Before, After } = require('cucumber');
+const sinon = require('sinon');
+const inquirer = require('inquirer');
+const { logger } = require('../../../lib/utils/logger');
+
+// These hooks run before each scenario, of every feature.
+
```

Comment:
In a weird way, it is nice that an import of this module is not needed in other steps modules.